packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -22,7 +22,8 @@
       background: var(--terra-consumer--mobile-nav-background, #383f42);
       box-shadow: 0 1px 3px;
       display: block;
-      opacity: 0.95;
+      opacity: 0.8;
+      width: 100vw;
```

Comment:
tablet is not supposed to have 100% screen width for the nav. remove these changes and i'll add the correct breakpoint changes with the other nav ui fixes.